docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc

```diff
@@ -67,24 +71,20 @@ projects:
         origin: "https://github.com/my-org/project1.git"
       checkoutFrom:
         revision: master           <1>
-      tag: 7.2.0
-      commitId: 36fe587
-      branch: master
 ----
-<1> `startPoint` is the general value for `tag`, `commitId`, and `branch`. The `startPoint`, `tag`, `commitId`, and `branch` parameters are mutually exclusive. When more than one is supplied, the following order is used: `startPoint`, `tag`, `commitId`, `branch`.
 ====
 
 `github`:: Same as `git` but for projects hosted on link:https://github.com/[GitHub] only. Use `git` for projects that do not use GitHub-specific features.
 
-`zip`:: Projects with sources in a ZIP archive. Location points to a ZIP file.
+`zip`:: Projects with sources in a ZIP archive. `checkoutFrom` points to a ZIP file.
```

Comment:
no checkoutFrom in Zip as per the schema https://docs.devfile.io/devfile/2.0.0/user-guide/api-reference.html